apps/frontend/src/views/Compare.vue

```diff
@@ -260,6 +267,17 @@ export default class Compare extends Vue {
     }
   ];
 
+  compareItems = [
+    'Run Time',
+    'Execution Length',
```

Comment:
This is the only instance of `Execution Length` in this PR. It seems to be called `Execution Time` below.

Also this should probably be called `Run Time`